src/pages/all-on-one/all-on-one.html

```diff
@@ -0,0 +1,232 @@
+<ion-header>
+  <mes-header></mes-header>
+  <report-header-v2></report-header-v2>
+</ion-header>
+
+<ion-content no-bounce>
```

Comment:
for some reason no-bounce isn't working for me

Reply:
I dont think that property works - https://github.com/ionic-team/ionic/issues/10329

Reply:
Is it bouncing for you? :)

Reply:
fixed